packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -82,6 +82,8 @@ export class DataStores implements IDisposable {
     // Stores the ids of new data stores between two GC runs. This is used to notify the garbage collector of new
     // root data stores that are added.
     private dataStoresSinceLastGC: string[] = [];
+    /** If true, throw an error when a tombstone data store is retrieved. */
+    private readonly shouldThrowOnTombstoneUsage: boolean;
```

Comment:
I wonder if `throwOnTombstoneUsage` is clear enough

Reply:
Renamed.